tools/container/container.go

```diff
@@ -36,13 +36,12 @@ const (
 
 // Options defines the shared options for the CLIs to configure containers runtimes.
 type Options struct {
-	Config        string
-	Socket        string
-	RuntimeName   string
-	RuntimeDir    string
-	SetAsDefault  bool
-	RestartMode   string
-	HostRootMount string
+	Config       string
+	Socket       string
+	RuntimeName  string
+	RuntimeDir   string
+	SetAsDefault bool
+	RestartMode  string
```

Comment:
Instead of making this change, we can set the `HostRoot` in this config struct in the same way we set the other defaults. This will also remove the unneeded function signature changes.

Reply:
Agreed. Thanks, it's a lot cleaner to stick with the current approach. I had thought I'd need to pass the `hostRoot`, the same way we do `toolkitRoot` since migrating from shell-based invocations to go API

Reply:
This is done